src/AltinnCore/Common/Services/Implementation/DataSI.cs

```diff
@@ -1,98 +1,186 @@
 using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Net.Http;
 using System.Threading.Tasks;
 using System.Xml.Serialization;
 using AltinnCore.Common.Configuration;
+using AltinnCore.Common.Helpers;
 using AltinnCore.Common.Models;
 using AltinnCore.Common.Services.Interfaces;
+using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Options;
 using Newtonsoft.Json;
 
 namespace AltinnCore.Common.Services.Implementation
 {
     /// <summary>
-    /// implementation for data handling
+    /// Service implementation for integration test
     /// </summary>
     public class DataSI : IData
     {
-        private readonly PlatformStorageSettings _platformStorageSettings;
+        private readonly ServiceRepositorySettings _settings;
+        private readonly GeneralSettings _generalSettings;
+        private readonly TestdataRepositorySettings _testdataRepositorySettings;
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private const string GetFormModelApiMethod = "GetFormModel";
+        private const string SaveFormModelApiMethod = "SaveFormModel";        
 
         /// <summary>
-        /// Initializes a new data of the <see cref="DataSI"/> class.
+        /// Initializes a new instance of the <see cref="FormSILocalDev"/> class.
```

Comment:
shouldnt this say DataSI not FormSILocalDev?

Reply:
fixed